bouncer/commands/run_test.ts

```diff
@@ -77,5 +77,5 @@ if (!matchingTestName) {
   console.error('No matching test function found');
   process.exit(1);
 } else {
-  execSync(`pnpm vitest --maxConcurrency=100 run -t "${matchingTestName}"`, { stdio: 'inherit' });
+  execSync(`pnpm vitest --maxConcurrency=500 run -t "${matchingTestName}"`, { stdio: 'inherit' });
```

Comment:
Setting the value this high causes Solana swaps to fail (I have discussed this with @MxmUrw , we think it's related to how engines process retries: if there is only one engine in the network, broadcasts are aborted much faster). 

I've reverted the concurrency back to 100. We can look at changing this selectively, maybe we can reduce it for Solana tests but keep it higher for other tests...